addons/im_livechat/tests/test_im_livechat_report.py

```diff
@@ -109,3 +110,49 @@ def test_redirect_to_form_from_pivot(self):
             "im_livechat_report_pivot_redirect_tour",
             login="operator_1",
         )
+
+    def test_day_of_week_ordering_week_start(self):
+        """Test that day of week grouping respects language week_start setting"""
+        operator = new_test_user(self.env, login="test_operator", groups="im_livechat.im_livechat_group_manager")
+        livechat_channel = self.env["im_livechat.channel"].create(
+            {"name": "Test Support", "user_ids": [operator.id]}
+        )
+        base_date = datetime.now() - timedelta(days=7)
```

Comment:
On next line, you assume `base_date` is sunday, which is incorrect (it depends on the day the test runs).